authenticate/management/commands/createusers.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management.base import BaseCommand
+from authenticate.models import CustomUser
+from django.utils.crypto import get_random_string
+
+class Command(BaseCommand):
+    help = 'Create n number of Users'
+
+    def add_arguments(self, parser):
+        parser.add_argument('users', type=int, help='Create n number of Users')
+
+    def handle(self, *args, **options):
+      
+      for i in range(options['users']):

```

Comment:
We should always use verbose name of variables even for temporary ones. Maybe use while

Reply:
Done